pyscriptjs/src/main.ts

```diff
@@ -204,9 +207,34 @@ export class PyScriptApp {
         // XXX: maybe the following calls could be parallelized, instead of
         // await()ing immediately. For now I'm using await to be 100%
         // compatible with the old behavior.
+        logger.info('importing pyscript');
+
+        // Save and load pyscript.py from FS
+        runtime.interpreter.FS.writeFile("pyscript.py", pyscript, { encoding: "utf8" });
+
+        // add `create_custom_element` and the current `PyScriptApp` to the Python namespace
+        // in the pyscript module scope
+        runtime.globals.set('create_custom_element', create_custom_element);
+        runtime.globals.set('_pyscript_app', this);
+
+        // inject it into the PyScript module scope
+        // TODO: Currently adding the imports for backwards compatibility, we should
+        //       probably remove it
+        await runtime.run(`
+        import pyscript
+        pyscript.create_custom_element = create_custom_element
+        pyscript._pyscript_app = _pyscript_app
+        pyscript.PyScript.set_version_info('${version}')
+        from pyscript import micropip, Element, console, document
```

Comment:
That's why I put the comment above... The intention here is to basically preserve what's in the current release, add a deprecation (cycle) warning and then remove it on the following release. 

On the matter of what should be in the global namespace I think we agreed already that we'll start only with `display`. We'll disagree on things to be added in the future, for sure, but I thought we had agreement for now.

About `display`... that's an EXCELLENT question. :)

Reply:
yes, that why I said +0. I just wanted to warn you about the future disagreements 😂

> About display... that's an EXCELLENT question. :)

I think we should investigate: it might be a hint that something very weird is going on